packages/core/content-type-builder/server/controllers/validation/types.js

```diff
@@ -107,7 +107,8 @@ const getTypeShape = (attribute, { modelType, attributes } = {}) => {
         regex: yup.string().test(isValidRegExpPattern),
       };
     }
-    case 'richtext': {
+    case 'richtext':
```

Comment:
I'm wondering watch minLength and maxLength will mean for a json field. I suspect just making part of the text bold for example will make it "longer", which is weird. But we can probably figure that out later

Reply:
I guess it could be the length in terms of number of blocks 🤷‍♂️ but true it doesn't make much sense.

Reply:
ok sounds good to me, I'll remove the min and max and just keep default and required